src/Juvix/Nets/Bohm.hs

```diff
@@ -288,43 +326,48 @@ fanInAux2 numFan (numOther, otherLang) level = do
   traverse_ linkAll [nodeOther1, nodeOther2, nodeFan1, nodeFan2]
   deleteRewire [numFan, numOther] [other1, other2, fanIn1, fanIn2]
 
-fanInAux0 ∷ (InfoNetwork net Lang m)
-          ⇒ Node → (Node, Primar) → m ()
-fanInAux0 numFan (numOther, otherLang) = do
-  sequentalStep
-  other1 ← newNode (Primar otherLang)
-  other2 ← newNode (Primar otherLang)
-  let nodeOther1 = RELAuxiliary0 { node       = other1
+fanInAux3 ∷ (InfoNetwork net Lang m)
+          ⇒ Node → (Node, Auxiliary3) → Int → m ()
+fanInAux3 numFan (numOther, otherLang) level = do
+  incGraphSizeStep 3
+  other1 ← newNode (Auxiliary3 otherLang)
+  other2 ← newNode (Auxiliary3 otherLang)
+  other3 ← newNode (Auxiliary3 otherLang)
```

Comment:
When fan in meets a node with 3 auxiliary ports, it should only duplicate the node once. Thus this logic needs to be reworked a bit